app/templates/components/code-module.hbs

```diff
@@ -21,6 +21,9 @@
         <button class="btn btn-primary btn-task inline" style="margin-top:20px;" {{action "save"}}>Uložit soubor na disk</button>
         <button class="btn btn-primary btn-task inline {{if (or running submitting) "disabled"}}" style="margin-top:20px;" {{action "reset"}}>Obnovit výchozí kód</button>
         {{input type="file" class="hide" id=(construct-id "load_input" module.id) accept=".py"}}
+        {{#if session.current_user.organisator}}
+            {{#link-to "admin/execs" (query-params module=module.id) class="btn btn-primary btn-task inline mtop20"}}→ Spuštění{{/link-to}}

```

Comment:
Nevím, zda je vizuálně zřetelný fakt, že tohle tlačítko není dostupné řešitelům. Mohlo by pak docházet k nedorozmněním mezi orgy a řešiteli. Nemohl by text tlačítka zahrnovat zmínku o tom, například "→ Spuštění [org]"
Nebo bychom mohli všechna orgovská tlačítka barevně odlišit, už takhle tam straší například orgovský report.

Reply:
Hmm, podle mě to není třeba, ale jestli to tam chceš, klidně to tam přidej.